webapp/src/types/mattermost-webapp/index.d.ts

```diff
@@ -1,5 +1,8 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
 export interface PluginRegistry {
```

Comment:
It will be great when we can import this from the webapp automatically 🎉

Reply:
I'm hoping to work on https://github.com/mattermost/mattermost/compare/master...hh_nov8-plugin-support again this Friday...